tests/apps/configurationapp/app.go

```diff
@@ -36,13 +37,17 @@ import (
 
 const (
 	daprGRPCPort               = 50001
-	configStore                = "configstore"
 	daprConfigurationURLStable = "http://localhost:3500/v1.0/configuration/"
 	daprConfigurationURLAlpha1 = "http://localhost:3500/v1.0-alpha1/configuration/"
 	separator                  = "||"
 	redisHost                  = "dapr-redis-master.dapr-tests.svc.cluster.local:6379"
 	writeTimeout               = 5 * time.Second
 	alpha1Endpoint             = "alpha1"
+	postgresComponent          = "postgres"
```

Comment:
Please, keep the test unchanged and switch between stores in the workflow config. See how we do it for Cosmosdb in state store.

Reply:
Done.